opal/runtime/help-opal-runtime.txt

```diff
@@ -57,3 +59,11 @@ WARNING: Cannot set both the MCA parameters opal_leave_pinned (a.k.a.,
 mpi_leave_pinned) and opal_leave_pinned_pipeline (a.k.a.,
 mpi_leave_pinned_pipeline) to "true".  Defaulting to mpi_leave_pinned
 ONLY.
+#
+[mpi-params:var_dump_color:format-error]
+WARNING: Token "%s" in opal_var_dump_color MCA parameter
+is not in the expected "key=value" format, ignoring.
+#
+[mpi-params:var_dump_color:unknown-key]
+WARNING: Key "%s" in opal_var_dump_color MCA parameter
+token "%s" is unknown, ignoring.
```

Comment:
Should we also list the valid keys in this error message?  E.g., include something like:

```
Valid keys are:

  %s
```

and the string for the `%s` is built up at run-time and passed in via the call to `opal_show_help()`.

Reply:
Sure, let's. The available valid keys are also in the param description, to which we could point the user when the error occurs, but might as well make it easier and repeat them here too.

Yes, I'd say let's go with `,` for consistency -- indeed the change is trivial. I'll get it done along with the new help text tomorrow.